src/lib.rs

```diff
@@ -1962,11 +1959,7 @@ where
     /// caller. If the length check fails, it returns `None`.
     #[inline]
```

Comment:
Since these are just wrappers around single function calls, I'd suggest going further and marking these as:
```suggestion
    #[inline(always)]
```

Reply:
Good call! Done here and elsewhere.